openmrs-19/src/main/java/org/motechproject/openmrs19/domain/Attribute.java

```diff
@@ -129,4 +171,28 @@ public void setAttributeType(AttributeType attributeType) {
     public void setLinks(List<Link> links) {
         this.links = links;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Can you stick with one conventation for equals and hashcode methods in domain classes ? Currently we are using Objects.equals and Objects.hash, moreover hashCode method should be before equals method.